app/modules/onboarding/assets/js/components/progress-bar/progress-bar.js

```diff
@@ -24,8 +24,14 @@ export default function ProgressBar() {
 		} );
 	}
 
-	progressBarItemsConfig.push(
-		{
+	if ( elementorAppConfig.onboarding.experiment ) {
```

Comment:
Just a note, we try to pass all the data we need from outside as props when mounting the component, so that the React apps are completely stand alone. This app is old so they did not follow that convention yet, but I just thought I'd mention it.

Reply:
Cool, Good to know!
Thanks :)